content-src/components/DiscoveryStreamComponents/SectionTitle/SectionTitle.jsx

```diff
@@ -1,22 +1,13 @@
 import React from "react";
 
-export class Topic extends React.PureComponent {
-  render() {
-    const {url, name} = this.props;
-    return (<li><a key={name} href={url}>{name}</a></li>);
-  }
-}
-
 export class SectionTitle extends React.PureComponent {
   render() {
-    const {topics} = this.props;
+    const {header: {title, subtitle}} = this.props;
     return (
-      <span className="ds-section-title">
-        <ul>
-          {topics && topics.map(t => <Topic key={t.name} url={t.url} name={t.name} />)}
-          <li><a className="ds-more-recommendations">More Recommendations</a></li>
-        </ul>
-      </span>
+      <div className="ds-section-title">
+        <div className="title">{title}</div>
+        {subtitle ? <div className="subtitle">{subtitle}</div> : null}
```

Comment:
Is it expected that section headers reuse this component or each will have their own styling? A quick glance I see Hero using ds-header and List using ds-list-title and neither support subtitles… The server seems to have a Header schema anyway… ?

Reply:
Not sure, they don't seem similar enough to me visually, but I cannot comment on list vs hero.

However, if there are slight differences, and keeping the components completely de coupled seems to make some sense to me as long as they are simple in complexity.

I think the only real value in reusability at the moment is in the ds-card for card grid and hero.